subgraph/src/KlerosCore.ts

```diff
@@ -126,10 +126,21 @@ export function handleNewPeriod(event: NewPeriod): void {
   }
 
   dispute.period = newPeriod;
-  dispute.lastPeriodChange = event.block.timestamp;
+  dispute.lastPeriodChangeTs = event.block.timestamp;
+  dispute.lastPeriodChangeBlockNumber = event.block.number;
+  let counter = PeriodIndexCounter.load(newPeriod);
```

Comment:
I think you can use `getAndIncrementCounter` here instead.

Reply:
fixed